Permissions for postdrop/postqueue aren't retained in our packaging. Address this
by setting permissions for both before we run postfix

diff --git a/pkg/docker/entrypoint.sh b/pkg/docker/entrypoint.sh
index aef32e7..4518a20 100755
--- a/pkg/docker/entrypoint.sh
+++ b/pkg/docker/entrypoint.sh
@@ -146,6 +146,7 @@ fi
 
 # Start Postfix to handle password resets etc.
 if [ -z "${PGADMIN_DISABLE_POSTFIX}" ]; then
+    sudo /usr/sbin/postdrop-permissions
     sudo /usr/sbin/postfix start
 fi
